dashboard/tests/functional/test_dashboard.py

```diff
@@ -131,6 +131,11 @@ def test_producttopuc_counts(self):
         self.assertIn('Products Linked To PUC', response,
                       'Where is the Products Linked to PUC card???')
         response_html = html.fromstring(response)
+
+        chem_count = response_html.xpath(
+            '/html/body/div[1]/div[2]/div[3]/div/div')[0].text
```

Comment:
```suggestion
            '//div[@class="card-body" and contains(h3, "Extracted Chemicals")]/div')[0].text
```
I've been playing around with XPath lately. This locator is a bit easier to understand and more robust if the layout changes.